actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -66,7 +66,21 @@ def call(env)
       response
     rescue Exception => exception
       raise exception unless request.show_exceptions?
-      render_exception(request, exception)
+
+      # If we got to this point, the only thing that can crash is the
```

Comment:
Think so too, wrote it before the commit. I'll drop it, the PR is descriptive enough.